activemodel/test/cases/secure_password_test.rb

```diff
@@ -62,14 +62,24 @@ class SecurePasswordTest < ActiveModel::TestCase
     assert_equal ["can’t be blank"], @user.errors[:password]
   end
 
-  test "create a new user with validation and password length greater than 72" do
+  test "create a new user with validation and password length greater than 72 characters" do
     @user.password = "a" * 73
     @user.password_confirmation = "a" * 73
     assert_not @user.valid?(:create), "user should be invalid"
     assert_equal 1, @user.errors.count
     assert_equal ["is too long (maximum is 72 characters)"], @user.errors[:password]
   end
 
+  test "create a new user with validation and password byte size greater than 72 bytes" do
+    # Create a password with 73 bytes by using a 3-byte Unicode character (e.g., "あ") 24 times, followed by a 1-byte character "a".
```

Comment:
Change character to an "a" under parenthetical

Reply:
The test needs to use multi-byte characters otherwise it won't demonstrate the extra check. The "a" character has a byte size of 1 in UTF-8 and all other ASCII-compatible encodings.